repose-aggregator/components/filters/keystone-v2/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Filter.scala

```diff
@@ -626,6 +626,7 @@ class KeystoneV2Filter @Inject()(configurationService: ConfigurationService,
       // Removes an extra slash at the end of the URI if applicable
       val serviceUri = keystoneV2Config.getIdentityService.getUri
       keystoneV2Config.getIdentityService.setUri(serviceUri.stripSuffix("/"))
+      akkaServiceClient.setConnectionPoolId(keystoneV2Config.getIdentityService.getConnectionPoolId)
```

Comment:
I believe I added some logging to the constructor to see if it was created once or multiple times.  I saw it created multiple times.  Although I'm doubting myself since this was 3 weeks ago.  This was a point of contention in my head, so I know I could be wrong.